src/main/java/com/xpanxion/java/springboot/da1/demo/controller/student3/BookStoreController3.java

```diff
@@ -0,0 +1,48 @@
+package com.xpanxion.java.springboot.da1.demo.controller.student3;
+
+import com.xpanxion.java.springboot.da1.demo.model.student3.response.BookStore;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.math.BigDecimal;
+import java.text.NumberFormat;
+import java.util.List;
+
+@RestController
+public class BookStoreController3 {
+
+    @Autowired
+    private JdbcTemplate template;
+
+    private final String SELECT_INVENTORY = """
+      select *
+      from bookstore_book bsb
+        join book b on b.book_id = bsb.book_id
+        join bookstore bs on bsb.bookstore_id = bs.bookstore_id 
+      where bs.bookstore_id = ?
+      """;
+
+   @GetMapping("student3/api/v1/bookstores/{bookstoreId}/books")
+   public List<BookStore> getInventory(@PathVariable int bookstoreId){
+       NumberFormat formatter = NumberFormat.getCurrencyInstance();
```

Comment:
It might be better to put the `NumberFormat` in the constructor of the `BookStore` model.  This separates things a bit better, separates the data access code from the formatting code.  So, for our project, the general rule would be: all currency formatting is done in the response models.